app/src/main/java/com/example/admin/lab_50_82/MainActivity.java

```diff
@@ -0,0 +1,64 @@
+package com.example.admin.lab_50_82;
+
+import android.content.Intent;
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+
+
+public class MainActivity extends ActionBarActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        // Button for launching activity for Lab #50
```

Comment:
Unnecessary comment. If you still want to specify behavior let's move it to field's javadoc.